package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "masters-way",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start:web": "pnpm --filter 'mw-webapp' run start",
+    "type-check:web": "pnpm --filter 'mw-webapp' type-check",
+    "test:web": "pnpm --filter 'mw-webapp' test",
+    "build-storybook:web": "pnpm --filter 'mw-webapp' build-storybook",
+    "build:web": "pnpm --filter 'mw-webapp' build",
+    "cypress:component:run:web": "pnpm --filter 'mw-webapp' cypress:component:run",
+    "cypress:e2e:run:web": "pnpm --filter 'mw-webapp' cypress:e2e:run"
```

Comment:
yeah, good idea, I can suggest to name it next way - `{moduleName}:{command}:{commandModeIfRequired}`

Reply:
done